ophyd/utils/epics_pvs.py

```diff
@@ -428,11 +428,12 @@ def _compare_maybe_enum(a, b, enums, atol, rtol):
 
 
 _type_map = {'number': (float, ),
-             'array': (np.ndarray, ),
+             'array': (np.ndarray, list),
```

Comment:
and I actually bet that on return this is going to be turned into an object array as we upcast everything that is 'dtype' array to `np.array`.

I should either revert these changes and use `ArrayAttributeSignal` or we make list a first class thing in the schema